src/window.rs

```diff
@@ -869,6 +882,26 @@ pub trait Window: AsAny + Send + Sync + fmt::Debug {
     /// - **Wayland / iOS / Android / Web:** Unsupported.
     fn is_visible(&self) -> Option<bool>;
 
+    /// Sets whether the window can be focused or not.
+    /// If `false`, the window can still accept mouse clicks but won't take keyboard focus on click.
+    /// This is useful for toolbar windows or accessibility tools such as on-screen keyboards.
+    ///
+    /// The default is `true`.
+    ///
+    /// ## Platform-specific
+    ///
+    /// - **Windows / macOS / X11:** Supported.
+    fn set_focusable(&self, focusable: bool);
+
+    /// Gets whether the window can be focused or not.
+    ///
+    /// See [`Window::set_focusable`] for details.
+    ///
+    /// ## Platform-specific
+    ///
+    /// - **Windows / macOS / X11:** Supported.
+    fn is_focusable(&self) -> Option<bool>;
```

Comment:
There's a clear default for this, isn't there? Return `true` when not supported (Winit windows are generally assumed to be focusable).

So why do we need the `Option`?

Reply:
removed